src/js/modules/accounts/components/forms/EditUserForm.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Field, reduxForm } from "redux-form";
+import injectSheet from "react-jss";
+
+const styles = {
+  successMessage: {
+    color: "green",
+  },
+  errorMessage: {
+    color: "red",
+  },
+};
+
+const validate = values => {
+  const errors = {};
+  if (
+    values.email &&
+    !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)
```

Comment:
Uhh, gonna need more explanation here. I guess that's a regex for checking emails, but you should probably assign it to a constant. Also, end the regex in an end string character

Reply:
that way emails like nick@nicholasyang.com.skynet.hackyou aren't allowed